src/main/java/uk/ac/ed/notify/job/OutboundEmailJob.java

```diff
@@ -0,0 +1,97 @@
+package uk.ac.ed.notify.job;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.quartz.DisallowConcurrentExecution;
+import org.quartz.Job;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.ac.ed.notify.channel.DeliveryAddress;
+import uk.ac.ed.notify.channel.DeliveryChannel;
+import uk.ac.ed.notify.entity.AuditActions;
+import uk.ac.ed.notify.entity.Notification;
+import uk.ac.ed.notify.entity.UserNotificationAudit;
+import uk.ac.ed.notify.repository.NotificationRepository;
+import uk.ac.ed.notify.repository.UserNotificationAuditRepository;
+import uk.ac.ed.notify.channel.DeliveryChannelRecipientStrategy;
+import uk.ac.ed.notify.service.OutboundEmailService;
+
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@DisallowConcurrentExecution
```

Comment:
Why can't emails be sent concurrently?

Reply:
The emails can -- in fact they are, by virtue of the `@Async` annotation on `OutboundEmailService.send()`.

It's the Quartz job that figures out what emails need sending (and updates the status to say they are sent) that I don't want to run more than one of at once.